src/charts/vertical_bar.coffee

```diff
@@ -191,6 +191,14 @@ Ember.Charts.VerticalBarComponent = Ember.Charts.ChartComponent.extend(
     _.uniq _.flatten(groups)
   .property 'groupedData.@each'
 
+  labelIDMapping: Ember.computed ->
+    @get('individualBarLabels').map (label, i) ->
+      labelLookup = {}
+      labelLookup[label] = i
+      labelLookup
+    .reduce(Ember.merge)
+  .property 'individualBarLabels'
```

Comment:
if an item is added or removed to `individualBarLabels`, `labelIDMapping` would not get updated